ci-operator/step-registry/ipi/install/vsphere/registry/ipi-install-vsphere-registry-commands.sh

```diff
@@ -27,17 +27,20 @@ oc wait --all --for=condition=Progressing=False clusteroperators.config.openshif
 
 oc wait --all --for=condition=Degraded=False clusteroperators.config.openshift.io --timeout=1m
 
-# Maps e2e images on dockerhub to locally hosted mirror
-if [[ "$JOB_NAME" == *"4.6-e2e"* ]]; then
-  echo "Remapping dockerhub e2e images to local mirror for 4.6 e2e vSphere jobs"
+function setMirrorMachineConfig() {
+  if [ -z $1 ]; then
+    CONFIGURE_MIRROR_ROLE="worker"
+  else
+    CONFIGURE_MIRROR_ROLE="$1"
+  fi
 
   oc create -f - <<EOF
 apiVersion: machineconfiguration.openshift.io/v1
 kind: MachineConfig
 metadata:
   labels:
-    machineconfiguration.openshift.io/role: worker
-  name: 98-e2e-registry-mirror
+    machineconfiguration.openshift.io/role: ${CONFIGURE_MIRROR_ROLE}
+  name: 98-e2e-registry-mirror-${CONFIGURE_MIRROR_ROLE}
```

Comment:
This is something for the future, but I wonder if we can use `/etc/containers/registries.conf.d/` instead of rewriting the registries.conf. That would probably allow us sideloading these manifests before install (to avoid `oc wait` commands)

Reply:
Yeah, that's a good point, i'll look in to that improvement.  I ran in to an issue while testing OKD on IBM with the samples operator that I'm addressing now.  The fix didn't behave quite as I expected but another fix is incoming.